stories/layout/report.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import React from 'react'
+import { Grid, Item, Layout } from '../../src'
+import { Box, loremIpsum } from '../core'
+import { colors1, colors2 } from '../core/stories.css'
+import styles from './layout.css'
+
+function P(props) {
+  return (
+    <Item size={12}>
+      <p {...props} />
+    </Item>
+  )
+}
+
+function Card({
+  children,
+  height = 70,
+}: {
+  children?: any,
+  height?: number,
+}): React$Element<any> {
+  return (
+    <Item size={12} style={{ height }}>
+      <div style={{ height: '100%' }} className={styles.card}>
+        <P>{children}</P>
+      </div>
+    </Item>
+  )
+}
+
+const text = (loremIpsum || '').substr(0, 250)
```

Comment:
jest was crashing, but you are right probably the better fix here is figure out why since it should be loading the file stub